lib/fathead/php/parse.js

```diff
@@ -163,4 +171,68 @@
     // CODE INITIALIZAITION
     main();
     //
+
+    /**
+     * Function responsible to extract examples information from a page
+     */
+    function buildExamples(examplesContainer, key) {
+
+        let examples, phpcode, exampĺeDescription, content = '';
+
+        if (!examplesContainer) {
+            // debug purposes: why dont have example?
+            _log('EXAMPLE NOT FOUND:', key);
+            return content;
+        }
+
+        // take just one, to avoid many info: https://github.com/duckduckgo/zeroclickinfo-fathead/issues/520#issuecomment-263280497
+        examples = examplesContainer.querySelector('.example, .informalexample');
+
+        if (!examples) {
+            // we dont have examples wrapper: need to look each case
+            _log('EXAMPLE WRAPPER NOT FOUND: ', key);
+            return content;
+        }
+
+        // get the first phpcode found
+        phpcode = examples.querySelector('.phpcode');
+
+
+        if (!phpcode) {
+            // we have examples section, but without phpcode
+            _log('EXAMPLE WITHOUT CODE: ', key)
+            return content;
+        }
+
+        // get the example description
+        exampleDescription = examples.querySelector('strong');
```

Comment:
looks like this needs `var` -- I'm seeing an error:

```
[F] Skipping : apc.setup.html
exampleDescription = examples.querySelector('strong');
                           ^
ReferenceError: exampleDescription is not defined
```

Reply:
fixed